src/components/UserActions/UserActions.scss

```diff
@@ -220,7 +220,8 @@ $timestamp-grey: #67768b;
             margin-left: 0.5em;
         }
 
-        .coveo-click {
+        .coveo-click,
+        .coveo-view {
```

Comment:
What's the reasoning behind this new rule in the context of SNOW-330?

Reply:
It fixes the issue by wrapping the hyperlink arround with the `word-wrap: break-word` rule and keeps the same style for the different hyperlinks type.
![image](https://user-images.githubusercontent.com/71288859/93533281-943c6380-f910-11ea-8458-039ff56d3927.png)